packages/site/index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <title>Atlantis</title>
-  <link rel="icon" type="image/svg+xml" href="./public/atlantis_favicon.svg" />
```

Comment:
After pulling in `master` I noticed this error 

![Screenshot 2024-12-23 at 1 53 57 PM](https://github.com/user-attachments/assets/4ba3706d-4a36-49d0-a340-7f864eb17695)